utils/ocr.ts

```
@@ -0,0 +1,36 @@
+    // Return a random mock text
+    const randomIndex = Math.floor(Math.random() * mockTexts.length);
+    return mockTexts[randomIndex];
+  }
+
+  static async extractTextFromDocument(documentUri: string): Promise<string> {
+    // Simulate processing time
+    await new Promise(resolve => setTimeout(resolve, 3000));
+    
+    return "This is extracted text from a document. In a real implementation, this would use OCR services to extract actual text from PDF or other document formats.";
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)